test/acceptance/authz/rbac_auto_no_permissions_test.go

```diff
@@ -41,8 +41,8 @@ func TestAuthzAllEndpointsNoPermissionDynamically(t *testing.T) {
 	defer cancel()
 
 	compose, err := docker.New().WithWeaviate().WithRBAC().
-		WithRbacUser("admin-User", adminKey, "admin-User").
-		WithRbacUser("custom-user", customKey, "custom").
+		WithRbacAdmin("admin-User", adminKey).
+		WithRbacUser("custom-user", customKey).
```

Comment:
**suggestion:** `withRbacUser()` has to be removed given that we will not support assigned non builtInRoles assignemnet

Reply:
The users still need to be added to the API key env vars

Reply:
reworked the user creation in tests and separeted them from RBAC